docs/synapse/httpapi.ipynb

```diff
@@ -339,6 +340,23 @@
     "        Each message has the following basic structure::\n",
     "\n",
     "            [ \"type\", { ..type specific info... } ]\n",
+    "            \n",
+    "        One particularly useful option that can be included in the request is a set of ``scrub`` rules.\n",
+    "        These rules describe what data should be included/excluded from the nodes that are returned.\n",
+    "        Currently the only rule type supported is ``include`` for ``tags``, but this list will be expanded\n",
+    "        over time. An example scrub rule to only include specific tags in the result output would look like::\n",
```

Comment:
"but this list will be expanded over time" is implied. Strike

Reply:
removed